### ISSUE TITLE
Add PHP `dom` extension to the `Contao system requirements` section

### ISSUE THREAD

**Issue by Serhii-DV** [contao/docs-archive#66]
While installing Contao 3.1.2 on new server just get next error:

```
Fatal error: Class 'DOMDocument' not found in /home/xxxx/www/xxxx.yyyyy.net/system/modules/core/library/Contao/System.php on line 602
```

**Comment by qzminski** [comment]
Nice catch ;-)

**Comment by Toflar** [comment]
Can you make a pull request on the English docs? I'll be happy to translate it into German for you :)

**Comment by aschempp** [comment]
Checked the core, this must have been added in 3.1

**Comment by tristanlins** [comment]
Doesn't tcpdf require DomDocument even in older Contao Versions?

**Comment by aschempp** [comment]
No idea, feel free to check the source code ;-)

**Comment by tristanlins** [comment]
I grepped a local 2.11.2 source for `domdocument`:

```
./system/modules/backend/Theme.php:195:                 $xml = new DOMDocument();
./system/modules/backend/Theme.php:368:                                 $xml = new DOMDocument();
./system/modules/backend/Theme.php:403:                 if (!$xml instanceof DOMDocument)
./system/modules/backend/Theme.php:613:         $xml = new DOMDocument('1.0', 'UTF-8');
./system/modules/backend/Theme.php:671:  * @param DOMDocument
./system/modules/backend/Theme.php:675: protected function addTableTlTheme(DOMDocument $xml, DOMElement $tables, Database_Result $objTheme)
./system/modules/backend/Theme.php:689:  * @param DOMDocument
./system/modules/backend/Theme.php:693: protected function addTableTlStyleSheet(DOMDocument $xml, DOMElement $tables, Database_Result $objTheme)
./system/modules/backend/Theme.php:735:  * @param DOMDocument
./system/modules/backend/Theme.php:739: protected function addTableTlModule(DOMDocument $xml, DOMElement $tables, Database_Result $objTheme)
./system/modules/backend/Theme.php:760:  * @param DOMDocument
./system/modules/backend/Theme.php:764: protected function addTableTlLayout(DOMDocument $xml, DOMElement $tables, Database_Result $objTheme)
./system/modules/backend/Theme.php:785:  * @param DOMDocument
./system/modules/backend/Theme.php:789: protected function addDataRow(DOMDocument $xml, DOMElement $table, Database_Result $objData)
```

I think even Contao 2.11 require dom document for certain functions.

**Comment by aschempp** [comment]
OK, I have changed the milestone then.

**Comment by aschempp** [comment]
@leofeyer should we update the system requirements on [contao.org](https://contao.org/en/manual/3.0/installation.html#configuring-the-live-server)?

**Comment by leofeyer** [comment]
Definitely. Will do.

**Comment by leofeyer** [comment]
I have updated the README file in contao/core@44bbd7efc5293fec2d67e50222030b1f547adb18. The documentation needs to be adjusted as well and while you're at it, please also check the other system requirements of the newer versions.

**Comment by aschempp** [comment]
Fixed in e2f10eb0bcdc612a12de1df5387b1960da93c225 and updated system requirements for Contao 3+ too.
